### PR TITLE
Allow to register custom TLD in `morph deploy`

### LINKED ISSUE #1727: Can't register contract with non-default TLD
## Expected Behavior
Deploy contract with `foo.bar` domain record in NNS
```
NNS: Set foo.bar -> 5e6de3a7677f4fa85acdb0341b3328d414e721e2
Waiting for transactions to persist...
```

## Current Behavior
Command throws an error
```
Error: `isAvailable`: invocation failed: at instruction 1246 (THROW): unhandled exception: "TLD not found"
```

## Possible Solution
It seems that smart contract expects an output but catches panic there:
https://github.com/nspcc-dev/neofs-node/blob/d57c57010fc8bdf9e1bcb1f22c3f04d2b51558e9/cmd/neofs-adm/internal/modules/morph/initialize_nns.go#L239-L246 
/cc @carpawell 

## Steps to Reproduce (for bugs)

1. Launch latest neofs-dev-env
2. Try to upload any smart-contract with `neofs-adm -c services/ir/neofs-adm.yml morph deploy --domain bar --contract foo/`

## Your Environment
neofs-adm v0.31.0
neofs-dev-env [c4a55445d1cb8f1aeacd00a9777119da731ef959](https://github.com/nspcc-dev/neofs-dev-env/commit/c4a55445d1cb8f1aeacd00a9777119da731ef959)

Related to #1629

### PR DESCRIPTION
**Pull request by fyrchik**
Close #1727.

### CODE REVIEW

**Review by alexvanin — DISMISSED** [review]
It works!

File: cmd/neofs-adm/internal/modules/morph/deploy.go [hunk 1]
```diff
@@ -109,13 +110,43 @@ func deployContractCmd(cmd *cobra.Command, args []string) error {
...
+
+			emit.AppCall(bw.BinWriter, nnsCs.Hash, "addRecord", callflag.All,
+				domain, int64(nns.TXT), cs.Hash.StringLE())
+		} else {
+			s, err := c.nnsRegisterDomainScript(nnsCs.Hash, cs.Hash, domain, false)
+			if err != nil {
+				return err
+			}
+			if len(s) != 0 {
+				c.Command.Printf("NNS: Set %s -> %s\n", domain, cs.Hash.StringLE())
```
[the remark this answers is not in the document]
Comment (fyrchik) [reply cmd/neofs-adm/internal/modules/morph/deploy.go]: Fixed

**Comment by fyrchik** [comment]
@alexvanin I have replaced stack with `INITSSLOT`, works for me, could you double check?

**Review by alexvanin — APPROVED** [review]
```
$ neofs-adm -c services/ir/neofs-adm.yml morph deploy --alphabet-wallets services/ir/ --contract example/ --domain foo
NNS: Set example.foo -> 5e6de3a7677f4fa85acdb0341b3328d414e721e2
Waiting for transactions to persist...
$
```

**Review by carpawell — APPROVED** [review]

**Pull request merged**